.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
you need to add your .gitignore file first thing before you add commit or push. You added all your node_modules to github